utils/geoutils.py

```diff
@@ -63,15 +64,18 @@ def clip_raster_with_gpkg(raster, gpkg, debug=False):
     """
     from shapely.geometry import box  # geopandas and shapely become a project dependency only during sample creation
     import geopandas as gpd
+    import fiona
     # Get extent of gpkg data with fiona
     with fiona.open(gpkg, 'r') as src:
+        gpkg_crs = src.crs
+        assert gpkg_crs == raster.crs
```

Comment:
maybe use `utils.verification.assert_crs_match(...)` instead?

as a few times I have noticed they dont always match 
or
I need to check src.crs['init'] instead

Reply:
... I realize src.crs['init'] opens up a whole new can of worms
so mb ignore for now ( I have put on my `# ToDo:` list )

Reply:
I will remove that assertion, it isn't meant to be there